setup.py

```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```

Comment:
It does make a large difference. If the original function would have been called, your code would bork. Now it won't.